src/test/kotlin/search/JumpSearchTest.kt

```diff
@@ -0,0 +1,41 @@
+package search
+
+import org.junit.Test
+
+class JumpSearchTest {
+
+    @Test
+    fun testBinarySearch1() {
```

Comment:
**suggestion (rename):** could we rename the tests `testJumpSearch`? (applies to all tests)